keep/README.md

```diff
@@ -0,0 +1,42 @@
+# Agent Check: Keep
+
+## Overview
+
+Keep is an open-source AIOps alert management tool.
+
```

Comment:
The overview should include: 

1. What does your product do (in 1-2 sentences)?
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data?